src/Core/Secrets/IConnectedModeSecrets.cs

```diff
@@ -0,0 +1,27 @@
+﻿/*
```

Comment:
I made a new folder "Secrets" in core hope thats oki

Reply:
I would have done the same, even though there is currently only one file in it.